desktop/src/main/java/bisq/desktop/main/funds/withdrawal/WithdrawalView.java

```diff
@@ -364,7 +364,7 @@ protected void activate() {
             feeToggleGroup.selectToggle(feeIncludedRadioButton);
 
         if (inputsToggleGroup.getSelectedToggle() == null)
-            inputsToggleGroup.selectToggle(useAllInputsRadioButton);
+            inputsToggleGroup.selectToggle(useAllInputs.get() ? useAllInputsRadioButton : useCustomInputsRadioButton);
```

Comment:
Legacy nit: current Bisq coding standards dictate use of curly braces for all conditional statements even one liners.

Reply:
I think it would be better to prepare all files fix in separated pull request. If you'd like I can do research regarding codacy configuration that would force developers to always use curly braces.

Reply:
> I think it would be better to prepare all files fix in separated pull request. If you'd like I can do research regarding codacy configuration that would force developers to always use curly braces.

I fear that would be quite a big PR to review 😬 - I'll check why Codacy didn't catch this in their checks.

Reply:
AvoidInlineConditionals of Checkstyle doesn't seem to be triggered by this 🤔. I don't remember anymore exactly, but I thought the ? : conditionals are still ok to be used. No strong opinion from my side on this.

Reply:
> I fear that would be quite a big PR to review

That's true. I think we can omit this for now and merge this pull request as it is.